sdk/storage/azure-storage-blob-cryptography/src/main/java/com/azure/storage/blob/specialized/cryptography/EncryptedBlobClientBuilder.java

```diff
@@ -406,7 +406,8 @@ public EncryptedBlobClientBuilder httpClient(HttpClient httpClient) {
     }
 
     /**
-     * Adds a pipeline policy to apply on each request sent.
+     * Adds a pipeline policy to apply on each request sent. The policy will be added after the retry policy. If
```

Comment:
```suggestion
     * Adds a {@link HttpPipelinePolicy} to apply on each request sent. The policy will be added after the retry policy. If
```

You may wish to name alias it with `{@link HttpPipelinePolicy pipeline policy}`